integrations/airflow/README.md

```diff
@@ -1,10 +1,5 @@
-## Moved
+# Marquez Airflow - `DEPRECATED`
 
-Airflow integration was moved to [OpenLineage](https://github.com/OpenLineage/OpenLineage/tree/main/integration/airflow).
-This module wraps OpenLineage codebase for backwards compatibility.
+[![Project status](https://img.shields.io/badge/status-deprecated-orange.svg)]()
 
-## Installation
-
-```bash
-$ pip3 install marquez-airflow
-```
+This library has been moved to [`OpenLineage/integration/airflow`](https://github.com/OpenLineage/OpenLineage/tree/main/integration/airflow). **The** `marquez-airflow` **library extends the class** [`openlineage.airflow.DAG`](https://github.com/OpenLineage/OpenLineage/blob/main/integration/airflow/openlineage/airflow/dag.py) **for backwards compatibility.**
```

Comment:
I would be more explicit with instructions like "use openlineage-airflow" and pointing to he installations instruction there.